app/javascript/components/components/TermCard.js

```diff
@@ -1,10 +1,44 @@
 import React, { Component, Fragment } from 'react'
 
 export class TermCard extends Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			form: {
+				term: '',
+				definition: '',
+			},
+		}
+	}
+
+	handleChange = (e) => {
```

Comment:
likely not need this handleChange function any more since functional props are now used